src/index.html

```diff
@@ -17,6 +17,101 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="logo_container"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="moyo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="container">
+          <li class="nav_list">
+            <a
+              href="#Apple"
+              id="Apple"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav_list">
+            <a
+              href="#Samsung"
+              id="Samsung"
+              class="nav_link color_regular"
```

Comment:
```suggestion
              class="nav_link"
```
redundant class
you should add all styles using nav_link selector. It already means there defined all default styles for the nav link. Then if you want to set another color in special cases you just write it for other selectors (is-active) below

Reply:
fixed